tests/Microsoft.Identity.Test.LabInfrastructure/Microsoft.Identity.Test.LabInfrastructure.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFrameworks>net461; netstandard2.0</TargetFrameworks>
```

Comment:
Could we keep .net471? According to the docs, https://docs.microsoft.com/en-us/dotnet/framework/install/on-windows-8-1, this is available for Win8.1 (not sure what this means, probably Win8 + Service Pack ?) but not for Win8

Reply:
I checked and the VM is 8.1, so i'll revert back on these.